### PR TITLE
Add a second per-frame callback that is invoked after the microtask queue is drained

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/flutter/issues/7626

### CODE REVIEW

**Comment by jason-simmons** [comment]
@abarth @Hixie

**Comment by abarth** [comment]
LGTM

File: lib/ui/hooks.dart [hunk 1]
```diff
@@ -83,6 +83,11 @@ void _beginFrame(int microseconds) {
     window.onBeginFrame(new Duration(microseconds: microseconds));
 }
 
+void _drawFrame() {
```
Comment (chinmaygarde) [inline lib/ui/hooks.dart]: Can this be called `_onDrawFrame`?

Comment (jason-simmons) [inline lib/ui/hooks.dart]: Named this _drawFrame for consistency with _beginFrame above.  Can change both if that's preferable.

**Comment by abarth** [comment]
I think you also need to add this to the list of externally reference dart names.

**Comment by abarth** [comment]
https://github.com/flutter/engine/blob/master/runtime/dart_vm_entry_points.txt

**Comment by jason-simmons** [comment]
done

**Comment by Hixie** [comment]
This is great. Let me know when you've rolled the engine for this and I'll update the framework.

**Comment by Hixie** [comment]
(unless you want to do that too, of course)

**Pull request merged**